src/webserver/flick/joints.py

```diff
@@ -0,0 +1,67 @@
+from flask import (
+    Blueprint, request, jsonify
+)
+from werkzeug.exceptions import abort
+from .db import get_db
+
+bp = Blueprint('joints', __name__)
+
+@bp.route('/joints')
+def get_joints():
+    db = get_db()
+    joints = db.execute(
+        'SELECT joint_id, name, location, description FROM joints'
```

Comment:
You can simply write `SELECT * FROM joints` to select all the fields.
Please make sure to change it in all queries.